LICENSE

```diff
@@ -268,6 +268,8 @@ For details, please see the files under: licenses/bin
 * asm-7.0.jar (BSD)
 * dnsjava-2.1.8.jar (BSD)
 * hamcrest-core-1.3.jar (BSD)
+* LatencyUtils-2.0.3.jar (BSD)
```

Comment:
BSD is an invalid (too broad) license reference. Please use SPDX license ID when possible: https://spdx.org/licenses/

Reply:
agreed

Reply:
[https://github.com/LatencyUtils/LatencyUtils/blob/master/LICENSE](LatencyUtils License) 
[https://github.com/HdrHistogram/HdrHistogram/blob/master/LICENSE.txt](HdrHistogram License)
These two licenses are both in BSD, what can I do to use SPDX License ? Thanks